<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OptimineExport</class>
 <widget class="QDialog" name="OptimineExport">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>596</width>
    <height>574</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Optimine DAT Exporter</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>490</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>59</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Full Scale:</string>
   </property>
   <property name="buddy">
    <cstring>fullScaleSpinBox</cstring>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>230</y>
     <width>59</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Zero Balance:</string>
   </property>
   <property name="buddy">
    <cstring>zeroBalanceSpinBox</cstring>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="fullScaleSpinBox">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>200</y>
     <width>66</width>
     <height>23</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
   <property name="decimals">
    <number>3</number>
   </property>
   <property name="singleStep">
    <double>0.010000000000000</double>
   </property>
   <property name="value">
    <double>10.013000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="zeroBalanceSpinBox">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>230</y>
     <width>66</width>
     <height>23</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
   <property name="decimals">
    <number>3</number>
   </property>
   <property name="minimum">
    <double>-99.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.010000000000000</double>
   </property>
   <property name="value">
    <double>0.013000000000000</double>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>20</y>
     <width>226</width>
     <height>52</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout" columnstretch="0,1,0">
    <item row="0" column="0">
     <widget class="QLabel" name="pressureFileLabel">
      <property name="text">
       <string>&amp;Pressure Data:</string>
      </property>
      <property name="buddy">
       <cstring>pressureLineEdit</cstring>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="pressureLineEdit"/>
    </item>
    <item row="0" column="2">
     <widget class="QToolButton" name="pressureFileButton">
      <property name="text">
       <string>...</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="acousticsFileLabel">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>&amp;Acoustics Data:</string>
      </property>
      <property name="buddy">
       <cstring>acousticsLineEdit</cstring>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="acousticsLineEdit">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QToolButton" name="acousticsFileButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>...</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>OptimineExport</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>438</x>
     <y>504</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>OptimineExport</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>506</x>
     <y>510</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
